policyengine_us/parameters/gov/usda/snap/temporary_local_benefit/dc/rate.yaml

```diff
@@ -0,0 +1,17 @@
+description: DC provides a temporary local SNAP benefit of this percentage of the maximum allotment.
+values:
+  2023-10-01: 0.1
+  # The temporary allotment expried in October 2024
+  2024-10-01: 0
+metadata:
+  period: year
+  unit: /1
+  label: DC temporary local SNAP benefit rate
+  reference:
+    - title: Supplemental Nutrition Assistance Program (SNAP) raise in District of Columbia in 2024.
+      href: https://dhs.dc.gov/page/give-snap-raise-heres-what-expect
+    - title: D.C. Law 24-301. Give SNAP a Raise Amendment Act of 2022. Sec 5084 amending (D.C. Official Code § 4-261.04 (b) ) 
+      href: https://code.dccouncil.gov/us/dc/council/laws/24-301
+    # Law was repealed 
```

Comment:
?

Reply:
§ 4–261.04 has been amended originally by [D.C. Law 24-301. Give SNAP a Raise Amendment Act of 2022](https://code.dccouncil.gov/us/dc/council/laws/24-301) to include the "give SNAP a raise program". That section has been since repealed and DC does not allow to see historic legal code versions, we still wanted to include the code section which was relevant

Reply:
That makes a lot of sense, I see how the wording was poor before, thanks!